runelite-api/src/main/java/net/runelite/api/Perspective.java

```diff
@@ -289,8 +289,8 @@ public static Polygon getCanvasTileAreaPoly(Client client, LocalPoint localLocat
 		int aoeSize = size / 2;
 
 		// Shift over one half tile as localLocation is the center point of the tile, and then shift the area size
```

Comment:
Does this comment need to be updated to reflect the code change?

Reply:
The comment still holds and the code speaks for itself here in my opinion.

Reply:
:-1: Adding one without explanation doesn't "speak for itself", in my opinion.